6-pubsub/src/PubSub/Worker.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace Google\Cloud\Samples\Bookshelf\PubSub;
+
+use Google\Cloud\PubSub\Subscription;
+
+class Worker
+{
+    const TOPIC_NAME = 'fill-book-details';
+    const SUB_NAME = 'book-worker-sub';
```

Comment:
Nit: What about moving them to yaml files or something? Not a big deal, but it will give us flexibility for tests.